packages/frontend/apps/mobile/src/app.tsx

```diff
@@ -67,12 +65,7 @@ export function App() {
       <FrameworkRoot framework={frameworkProvider}>
         <I18nProvider>
           <AffineContext store={getCurrentStore()}>
-            <Telemetry />
-            <RouterProvider
-              fallbackElement={<AppFallback />}
```

Comment:
AppFallback is design for desktop, need a new one

Reply:
Should have to resolve conflict soon 😄 #8686